hummingbot/connector/gateway_EVM_AMM.py

```diff
@@ -188,12 +198,20 @@ def is_pending_approval(self, token: str) -> bool:
         ]
         return True if token in pending_approval_tokens else False
 
+    def update_transaction_fees(self, transaction_fee_token: str, amount: Decimal):
+        """
+        Records the last known transaction fees for a trade on the DEX.
+        """
+        self._last_known_network_transaction_fee = TokenAmount(transaction_fee_token, amount)
+
```

Comment:
I believe we can simply use the `@property.setter` here.

```suggestion
    @network_transaction_fee.setter
    def network_transaction_fee(self, token_amount: TokenAmount):
        """
        Records the last known transaction fees for a trade on the DEX.
        """
        self._last_known_network_transaction_fee = token_amount

```